src/identifiers.md

```diff
@@ -18,7 +18,7 @@ Identifiers follow the specification in [Unicode Standard Annex #31][UAX31] for
 * `foo`
 * `_identifier`
 * `r#true`
-* `Москва`
+* `Київ`
```

Comment:
Rather than replacing Moscow with Kyiv, I think it should be replaced with an entirely neutral term.

Reply:
ok, lets make it "Lemon"